geode-core/src/main/java/org/apache/geode/distributed/internal/DistributionAdvisor.java

```diff
@@ -1319,7 +1313,8 @@ public Set adviseProfileRemove() {
   // must synchronize when modifying profile array
   private synchronized boolean basicAddProfile(Profile p) {
     // don't add more than once, but replace existing profile
-    profilesVersion++;
+    // try {
```

Comment:
Looks like leftovers from debugging.

Reply:
This is just a straight revert command.  I'm trying not to cause any confusion with git and/or review. If I make modifications, I think this will cause confusion for anyone tracing this in the future.